Localization/localization.cfg

```diff
@@ -131,135 +131,135 @@ Localization
+        precisemaneuver_orbittools_turnup_tooltip       = Agregar una órbita
+        precisemaneuver_orbittools_turndown_tooltip     = Reducir una órbita
+        precisemaneuver_orbittools_circularize_tooltip  = Circularizar la órbita con la maniobra actual
+        precisemaneuver_orbittools_copy_tooltip         = Copiar detalles de maniobra al copiapapeles
+        precisemaneuver_orbittools_paste_tooltip        = Copiar maniobra del copiapapeles\n(puedes copiarla desde el modTransfer Window Planner)
         
-        precisemaneuver_gizmo_undo                      = Undo
-        precisemaneuver_gizmo_redo                      = Redo
-        precisemaneuver_gizmo_sensitivity               = Sensitivity
+        precisemaneuver_gizmo_undo                      = Deshacer
+        precisemaneuver_gizmo_redo                      = Rehacer
+        precisemaneuver_gizmo_sensitivity               = Sensibilidad
         precisemaneuver_gizmo_Ap                        = Ap
         precisemaneuver_gizmo_Pe                        = Pe
                 
-        precisemaneuver_encounter                       = Next Enc.
+        precisemaneuver_encounter                       = Próximo Enc.
         precisemaneuver_encounter_PE                    = PE
                 
-        precisemaneuver_ejection_angle                  = Ej. Angle
-        precisemaneuver_ejection_angle_format           = ｢0:0.00° from prograde;0.00° from retrograde｣
-        precisemaneuver_ejection_inclination            = Ej. Incl.
-        precisemaneuver_ejection_inclination_format     = ｢0:0.00° north;0.00° south｣
+        precisemaneuver_ejection_angle                  = Ángulo de Eyec.
+        precisemaneuver_ejection_angle_format           = ｢0:0.00° desde progrado;0.00° desde retrogrado｣
+        precisemaneuver_ejection_inclination            = Incl. Eyec.
+        precisemaneuver_ejection_inclination_format     = ｢0:0.00° norte;0.00° sur｣
         
         precisemaneuver_info_ap                         = Apoapsis
         precisemaneuver_info_pe                         = Periapsis
         precisemaneuver_info_inclination                = Incl.
         precisemaneuver_info_eccentricity               = e
                 
-        precisemaneuver_conics                          = Conics\ndraw mode
-        precisemaneuver_conics_more                     = More Patches
-        precisemaneuver_conics_more_tooltip             = Shows more conic patches in mapview
-        precisemaneuver_conics_less                     = Fewer Patches
-        precisemaneuver_conics_less_tooltip             = Shows fewer conic patches in mapview
+        precisemaneuver_conics                          = Conos\nmodo de dibujo
+        precisemaneuver_conics_more                     = Más parches
+        precisemaneuver_conics_more_tooltip             = Mostrar más parches de conos en la vista de mapa
```

Comment:
wrong again "Mostrar mas trayectorias en la vista mapa" or could be "Mostrar mas orbitas en la vista mapa"